gadwords.php

```diff
@@ -19,140 +19,142 @@
-
-		$this->context->smarty->assign(array(
-			'module_dir' => $this->_path,
-			'code' => $code,
-			'landing_page' => $landing_page,
-			'is_local' => $is_local,
-		));
-		return $this->display(__FILE__, 'views/templates/admin/gadwords.tpl');
-	}
-
+    const PRESTASHOP_ADS_MODULE_NAME = 'emarketing';
+    const PRESTASHOP_ADS_MODULE_ID = 18716;
+
+    public $name;
+    public $tab;
+    public $version;
+    public $author;
+    public $bootstrap;
+    public $displayName;
+    public $description;
+    public $ps_versions_compliancy;
+    public $isPrestashop16;
+
+    public function __construct()
+    {
+        $this->name = 'gadwords';
+        $this->tab = 'advertising_marketing';
+        $this->version = '2.0.0';
+        $this->author = 'PrestaShop';
+        $this->bootstrap = true;
+        parent::__construct();
+
+        $this->displayName = $this->l('Google AdWords');
+        $this->description = $this->l('You want to be more visible on Google and attract new clients ? Use our 75€ promo code on Google Adwords !');
+        $this->ps_versions_compliancy = array('min' => '1.6', 'max' => _PS_VERSION_);
+        $this->isPrestashop16 = version_compare(_PS_VERSION_, '1.7.0.0', '<');
+        $this->controller = 'AdminModuleManager';
+    }
+
+    /**
+     * Install the module gadwords and manage the module Prestashop Ads
+     *
+     * @return bool
+     */
+    public function install()
+    {
+        return parent::install() & $this->installTab();
+    }
+
+    /**
+     * This method is often use to create an ajax controller
+     *
+     * @param none
+     * @return bool
+     */
+    private function installTab()
+    {
+        $tab = new Tab();
+        $tab->class_name = $this->controller;
+        $tab->active = 1;
+        $tab->name = array();
+
+        foreach (Language::getLanguages(true) as $lang) {
+            $tab->name[$lang['id_lang']] = $this->name;
+        }
+
+        $tab->id_parent = -1;
+        $tab->module = $this->name;
+
+        return $tab->add();
+    }
+
+    /**
+     * Load Assets
+     *
+     * @return void
+     */
+    private function loadAssets()
+    {
+        if ($this->isPrestashop16) {
+            $this->context->controller->addCSS('//fonts.googleapis.com/icon?family=Material+Icons');
+        }
+
+        $this->context->controller->addCSS($this->_path.'views/css/gadwords.css');
+        $this->context->controller->addJS($this->_path.'views/js/admin.js');
+
+        Media::addJsDef(array(
+            'gadwords_controller_url' => $this->context->link->getAdminLink($this->controller),
+            'gadwords_controller_name' => $this->controller,
+        ));
+    }
+
+    /**
+     * Get the module's configuration link for Prestashop 1.6 or Prestashop 1.7
+     *
+     * @param  bool $moduleIsInstalled
+     *
+     * @return string
+     */
+    public function getModulePrestashopAdsLink(bool $moduleIsInstalled)
```

Comment:
```suggestion
    public function getModulePrestashopAdsLink($moduleIsInstalled)
```

Reply:
Thank you , I removed the bool type for the PHP5 compatibility